alkira/resource_alkira_policy_rule.go

```diff
@@ -105,13 +105,15 @@ func resourceAlkiraPolicyRule() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{"ALLOW", "DROP"}, false),
 			},
 			"rule_action_service_types": {
-				Description: "Based on the service type, traffic is routed to service of the given type.",
+				Description: "Based on the service type, traffic is routed to service of the given type." + 
```

Comment:
Add a space at the end like this `type. "`, so the generated doc will have a space.